sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -129,7 +130,7 @@ def load_provider(*args, **kwargs) -> "AzureAppConfigurationProvider":
                 provider._dict[trimmed_key] = secret
             elif isinstance(config, FeatureFlagConfigurationSetting):
                 feature_management = provider._dict.get(FEATURE_MANAGEMENT_KEY, {})
-                feature_management[trimmed_key] = config.value
+                feature_management[trimmed_key[len(FEATURE_FLAG_PREFIX) :]] = config.value
```

Comment:
len(FEATURE_FLAG_PREFIX) is the length of the string. Wanted to double check if this is the design.

Reply:
Looks like you want to remove the prefix ".appconfig.featureflag/". 

Do you need to check if it starts with it?

Reply:
It "should" always have it. But added a check to be safe.